.github/ISSUE_TEMPLATE/documentation_request_correction.yml

```diff
@@ -0,0 +1,70 @@
+# SPDX-FileCopyrightText: Copyright (c) 2022 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Documentation - Correction/Update Request
+description: Request corrections or updates to existing cuSpatial documentation
+title: "[DOC]: "
+labels: ["doc", "? - Needs Triage"]
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to help cuSpatial and improve our documentation!
+
+  - type: dropdown
+    id: criticality
+    attributes:
+      label: How would you describe the priority of this documentation request
+      options:
+        - Critical (currently preventing usage)
+        - High
+        - Medium
+        - Low (would be nice)
```

Comment:
Is it helpful to have two levels between lowest and highest? That's kind of like two mediums.  Perhaps only have one medium, and give it a guideline like you have for highest and lowest.

Reply:
I think it's helpful because the bar we pose for critical is so high. My concern with only 3 options is so much will fall into 'medium' and I wanted to give people a minor amount of granularity between Low and Critical because this isn't cuSpatial's decision of priority, it's how the reporter _feels_ about the priority.

I struggle with good (descriptions) for the high and medium, but I also kind of like it blank since it lets the person submitting the issue make that decision themselves. We bound the upper and lower values, and the middle ones are squishy by design.

@thomcom @isVoid - thoughts?

Reply:
I guess I prefer two options high and low, mostly just because of decision-making pressure when working through these forms.